docker/third-party-programs.txt

```diff
@@ -0,0 +1,1403 @@
+Intel(R) oneAPI Toolkits Third Party Programs File
```

Comment:
what's this file for?

Reply:
It is needed if we are to pull from Intel dockerfile directly. If we want to get rid of it we may need to explcitly add each itel dockerfile command and remove the command needed for this file...

In other words: we pull from this file, and need to handle this line somehow: https://github.com/intel/oneapi-containers/blob/master/images/docker/hpckit/Dockerfile.ubuntu-18.04#L35